eng/scripts/test-matrix/test-matrix-functions.ps1

```diff
@@ -0,0 +1,317 @@
+Set-StrictMode -Version "4.0"
+
+function CreateDisplayName([string]$parameter, [hashtable]$displayNames) {
```

Comment:
I cannot remember if I gave you this feedback already or not but I generally like to put `{` on its own line for larger blocks like a function.

Reply:
You did, I just misunderstood it :)